cmake/std/PullRequestLinuxDriverTest.py

```diff
@@ -0,0 +1,483 @@
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Parse the repo and build information')
+    parser.add_argument('sourceRepo',
+                        help='Repo with the new changes',
+                        action='store')
+    parser.add_argument('sourceBranch',
+                        help='Branch with the new changes',
+                        action='store')
+    parser.add_argument('targetRepo',
+                        help='Repo to merge into',
+                        action='store')
+    parser.add_argument('targetBranch',
+                        help='Branch to merge to',
+                        action='store')
+    parser.add_argument('job_base_name',
+                        help='The jenkins job base name')
+    parser.add_argument('github_pr_number',
+                        help='The github PR number')
+    parser.add_argument('job_number',
+                        help='The jenkins build number')
+    parser.add_argument('workspaceDir',
+                        help='The local workspace directory jenkins set up')
+    arguments = parser.parse_args()
+    return arguments
+
+
+def print_input_variables(arguments):
+    print(
+        "\n==========================================================================================",
+        file=sys.stdout)
+    print("Jenkins Input Variables:", file=sys.stdout)
+    print("- JOB_BASE_NAME: {job_base_name}".format(
+        job_base_name=arguments.job_base_name),
+          file=sys.stdout)
+    print("- WORKSPACE    : {workspace}".format(
+        workspace=arguments.workspaceDir),
+          file=sys.stdout)
+    print(
+        "\n==========================================================================================",
+        file=sys.stdout)
+    print("Parameters:", file=sys.stdout)
+    print("- TRILINOS_SOURCE_REPO  : {source_repo}".format(
+        source_repo=arguments.sourceRepo),
+          file=sys.stdout)
+    print("- TRILINOS_SOURCE_BRANCH: {source_branch}\n".format(
+        source_branch=arguments.sourceBranch),
+          file=sys.stdout)
+    print("- TRILINOS_TARGET_REPO  : {target_repo}".format(
+        target_repo=arguments.targetRepo),
+          file=sys.stdout)
+    print("- TRILINOS_TARGET_BRANCH: {target_branch}\n".format(
+        target_branch=arguments.targetBranch),
+          file=sys.stdout)
+    print("- PULLREQUESTNUM        : {num}".format(
+        num=arguments.github_pr_number),
+          file=sys.stdout)
+    print("- BUILD_NUMBER          : {num}".format(
+        num=arguments.job_number),
+          file=sys.stdout)
+    print(
+        "\n==========================================================================================",
+        file=sys.stdout)
+
+
+def confirmGitVersion():
+    git_version_string = subprocess.check_output(['git', '--version'])
+    if git_version_string[12:16] != "2.10":
+        raise SystemExit("Git version  should be 2.10 or better - Exiting!")
```

Comment:
The output says git 2.10 or better is acceptable, but the check is only for git 2.10.

Reply:
I found that odd when I copied it over. Lexical comparison can be a bit difficult, but now is the time.

Reply:
Done